packages/salesforcedx-vscode-core/package.json

```diff
@@ -625,6 +632,14 @@
       {
         "command": "sfdx.internal.lightning.lwc.create",
         "title": "%force_lightning_lwc_create_text%"
+      },
+      {
+        "command": "sfdx.force.metadata.view.refresh",
+        "title": "Refresh tests",
```

Comment:
This needs to be a label and the text should be something along the lines of `Refresh Org Browser`. cc @hbc-bindu

Reply:
Label looks good to me.